pages/dkp/konvoy/2.2/choose-infrastructure/eks/advanced/new/index.md

```diff
@@ -50,7 +50,7 @@ Before you start, make sure you have completed the steps in [Bootstrap][bootstra
 1.  Create the cluster from the objects.
 
     ```bash
-    kubectl create -f ${CLUSTER_NAME}.yaml
+    dkp create cluster aks --cluster-name=${CLUSTER_NAME} --additional-tags=owner=$(whoami)
```

Comment:
should this be `eks` ?
```suggestion
    dkp create cluster eks --cluster-name=${CLUSTER_NAME} --additional-tags=owner=$(whoami)
```

Reply:
I'm trying to remember - I think it might have been right with the kubectl create, @gflorinskaya @dkoshkin  - is this right? I think that if you kubectl apply, well, create, this cluster_name.yaml file, it just applies whatever you made?